registration/include/pcl/registration/impl/ndt.hpp

```diff
@@ -427,22 +430,20 @@ NormalDistributionsTransform<PointSource, PointTarget>::computeHessian (Eigen::M
     {
       cell = *neighborhood_it;
 
-      {
-        x_pt = input_->points[idx];
-        x = Eigen::Vector3d (x_pt.x, x_pt.y, x_pt.z);
+      x_pt = input_->points[idx];
+      x = Eigen::Vector3d (x_pt.x, x_pt.y, x_pt.z);
 
-        x_trans = Eigen::Vector3d (x_trans_pt.x, x_trans_pt.y, x_trans_pt.z);
+      x_trans = Eigen::Vector3d (x_trans_pt.x, x_trans_pt.y, x_trans_pt.z);
```

Comment:
Can the scope of `x` and `x_trans` be reduced to the body of this foor loop?

Reply:
I reduced the scope of them.
I saw many variables with too large scope in the original implementation (e.g., ```x_pt``` and ```x_trans_pt``` at L185). Should I fix them as well?

Reply:
If you don't mind it would be awesome, but it's not required.

Reply:
I reduced variable scopes through the code.